unfetter-discover/unfetter-ui
# Display and initial setup of baseline settings

Fixes unfetter-discover/unfetter#928

Introduction of baseline settings panel and initial launch into baselines wizard, which is currently empty.  Next step is the category builder panel.

Note:  `beta` header for assessments 3.0 will go into my next PR.

## To test:
1. Pull this PR
2. Restart unfetter-ui:   `docker-compose restart unfetter-ui`
3. Navigate to Assessments 3.0
4. Click down triangle icon to display existing baselines
5. Click 'Create' button
6. Enter baseline title, description, and select org; click Continue
7. Initial baselines wizard should display, with "Category" and "Summary" steps listed on the left sidebar
